include/ext/string_view.hpp

```diff
@@ -17,14 +17,13 @@
 #ifndef EXT_STRING_VIEW_HPP
 #define EXT_STRING_VIEW_HPP
 
-#include <boost/utility/string_view.hpp>
+#include <string_view>
 
 namespace ext {
 
-// TODO: the following 'using' directives can be changed to
-//   using std::string_view;
-// once MySQL source code switches to c++17
-using boost::string_view;
+// this using declaration left only for compatibility reasons
```

Comment:
Could you please elaborate regarding compatibility reasons?
AFAICS you have removed all references to ext::string_view in our tree, so why not to remove this header completely?
Or are you worried about 3rd-party code who might be using this header? How likely is that? If yes, then IMO it is better to explicitly mention this code in the comment.

Reply:
Comment updated

Reply:
OK.